olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/StreamLayerClient.h

```diff
@@ -86,8 +94,35 @@ class DATASERVICE_READ_API StreamLayerClient final {
 
   ~StreamLayerClient();
 
+  /**
+   * @brief Cancels all active and pending requests.
+   * @return True on success.
+   */
+  bool CancelPendingRequests();
```

Comment:
```suggestion
  void CancelPendingRequests();
```
Better to return `void`, cause `PendingRequests::CancelAllAndWait()` always return true.

Reply:
Ok, discussed offline - actually better to be consistent with other dataservice-read clients and leave it with bool. @mykhailo-kuchma , please confirm to leave the signature's return type with bool or change it to 'void'.